spec/factories/scheme.rb

```diff
@@ -19,6 +20,7 @@
       scheme_type { 7 }
       intended_stay { "M" }
       primary_client_group { "G" }
+      has_other_client_group { 1 }
```

Comment:
I would have expected that the attributes defined outside of traits are defaults that would still be applied to all factory instances so this line would be redundant.

If you remove it and everything still works, you can probably remove line 18 as well `registered_under_care_act`

Reply:
I could add a trait for a separate scheme that has a secondary client group and test separately but I don't personally see a benefit in that when I'm doing that test anyway more explicitly in the specs